### ISSUE TITLE
Style break by 3.0.0 due to html encoding

### ISSUE THREAD

**Issue by xibman** [Faire/mjml-react#69]
Hi everyone,

Great work on this fork to continue this project.

I see a breaking change with the last version on styles :

```jsx
<MjmlStyle>{`
          .main-body > div{background:#fff;}
`}</MjmlStyle>
```

The superior is encoded into html entities to `&#x3E;`. 
So the css end working.

The MjmlStyle content must not be encoded to be readable by browser and email client

**Comment by emmclaughlin** [comment]
Hi @xibman! We should have documented this update better. In the previous version (V2) `MjmlStyle` was automatically passing its children through `dangerouslySetInnerHtml` under the hood. We felt like it wasn't reasonable using `dangerouslySetInnerHtml` under the hood where developers may not know. So we updated and the children here are sanitized by React, but we added the `dangerouslySetInnerHtml` prop directly to MjmlStyle. So if you pass your string through the [dangerouslySetInnerHtml](https://github.com/Faire/mjml-react/blob/main/src/mjml/MjmlStyle.tsx#L12) prop no encoding should occur. 

As always, happy to hear any opinions on the change 🙂
